joomla/joomla-cms
# [com_menus] menus acl: don't allow to delete if user doesn't have permission to delete in that menu type

#### Summary of Changes

Some ACL in the menus list view aren't working quite right:
- If the user can't delete a particular menu type or is menu items it shouldn't be allowed to delete the menu type.
#### Testing Instructions
1. Create a user groups "testgroup" (child of "Administrator" user group)
2. Go to Menus -> Manager, create a "menu-dont-delete" menu type and set the "testgroup" group to have no permission (Denied) to "Delete"
3. Create a user "testuser" and add it only to "testgroup"
4. Open a new private browser window and login to backend with "testuser".
5. Go to menu Menus -> Manage
6. Try to delete the "menu-dont-delete". The menu is deleted!
   
   It shouldn't since you removed that permission in "2."
7. Apply patch. Repeat step 2. and 6. all good, a error message will appear.
   You can also try with several menus at the same time.

@bembelimen @infograf768 please test
